components/pre_trial_pricing_modal.tsx/index.tsx

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+import {Modal} from 'react-bootstrap';
+
+import Content from './content';
+
+import './pre_trial_pricing_modal.scss';
+
+function PreTrialPricingModal() {
+    const [show, setShow] = React.useState(true);
+    return (
+        <Modal
+            className='PreTrialPricingModal'
+            show={show}
+            id='preTrialPricingModal'
+            onExited={() => {
+                setShow(true);
+            }}
+            data-testid='preTrialPricingModal'
+            dialogClassName='a11y__modal'
+            onHide={() => {
+                setShow(false);
+            }}
```

Comment:
This seems to not be working as it never closes for me:


https://user-images.githubusercontent.com/936315/167966766-1fc34f4d-0d5d-43b8-8807-efee6c779584.mov